### PR TITLE
[Fix](Create View) support create view from tvf

### PR DESCRIPTION
**Pull request by BePPPower**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Support `create view as select * from tvf()`

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/correctness_p0/table_valued_function/test_create_view_from_tvf.groovy [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    sql """ CREATE VIEW ${testViewName}
+            (
+                k1 COMMENT "number"
+            )
+            COMMENT "my first view"
+            AS
+            SELECT number as k1 FROM numbers("number" = "10");
+        """
+
+    qt_view1 """ select * from ${testViewName} """
```
Comment (morningman) [inline regression-test/suites/correctness_p0/table_valued_function/test_create_view_from_tvf.groovy:30]: add more complex cases, such as:
1. create view without column definition
2. create view with join and where

Comment (BePPPower) [reply regression-test/suites/correctness_p0/table_valued_function/test_create_view_from_tvf.groovy]: ok

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**